app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -94,6 +94,8 @@
   .grid {
     margin-left: 0;
     margin-right: 0;
```

Comment:
The grid is restricted to 960px in honeycrisp, but my expectation when there are css classes like "width-one-half" is that the grid would be full width of its parent and then you can restrict that width if necessary by adding a wrapper class to the parent with a max width. For the purposes of our app where the default is full-width pages, unsetting the inherited max-width makes our width-one-half and width-two-thirds classes truly 1/2 and 2/3 of the page respectively.